packages/web-app-files/src/App.vue

```diff
@@ -55,7 +53,8 @@ export default defineComponent({
   },
   created() {
     this.$root.$on('upload-end', () => {
-      this.delayForScreenreader(() => this.$refs.filesListWrapper.focus())
+      // delay for screen reader virtual buffer
+      setTimeout(() => this.$refs.filesListWrapper.focus(), 500)
```

Comment:
fyi: the `delayForScreenreader` mixin was only used here, so I deleted the method